### DISCUSSION TITLE
SSO: Cognito

### DISCUSSION THREAD

**Discussion by theluk** [langfuse/langfuse#1776] [Ideas]
### Describe the feature or potential improvement

Would be open for implementation

https://next-auth.js.org/providers/cognito

**Comment by marcklingen** [comment]
Hi @theluk, can you contribute this and test that it works end-to-end as expected? Happy to help and get this merged quickly as it is a very simple addition

**Comment by marcklingen** [comment]
Cognito is supported by langfuse: https://langfuse.com/docs/deployment/self-host#sso

Thanks again for raising this here
